node/libs/schema/src/std_conv.rs

```diff
@@ -80,3 +80,24 @@ impl ProtoFmt for time::Duration {
         }
     }
 }
+
+impl ProtoFmt for bit_vec::BitVec {
+    type Proto = proto::BitVector;
+
+    fn read(r: &Self::Proto) -> anyhow::Result<Self> {
+        let size = *required(&r.size).context("size")? as usize;
+        let mut this = Self::from_bytes(required(&r.vector).context("vector")?);
+        if this.len() < size {
+            anyhow::bail!("'vector' has less than 'size' bits");
+        }
+        this.truncate(size);
+        Ok(this)
+    }
+
+    fn build(&self) -> Self::Proto {
+        Self::Proto {
+            size: Some(self.len() as u64),
+            vector: Some(self.to_bytes()),
```

Comment:
Bikeshedding: Maybe, use `bytes` instead of `vector`? IMO, "vector" looks quite ambiguous.

Reply:
"bytes" is a reserved word in protobuf syntax